core/services/src/main/java/org/exoplatform/services/cms/documents/impl/DocumentServiceImpl.java

```diff
@@ -828,6 +828,7 @@ public long getEditorsIdleTimeout() {
          + "or exo:fileType like '%odt%' "
          + "or exo:fileType like '%officedocument%' "
          + "or exo:fileType like '%ms%') "
+         + "and not (exo:dlp = 'dlpsymlink' )"
```

Comment:
```suggestion
         + "and not (jcr:primaryType = 'exo:exoDlpItem' )"
```
Thus I'm not sure that we need a specific property. The mixin should be enough